apps/web/src/utils/charts.ts

```diff
@@ -105,3 +107,16 @@ export function buildTimeSeriesOptions({
     grid: yUnit ? YUINT_TO_GRID[yUnit] : undefined,
   };
 }
+
+export function useArrayBestUnit(arr?: number[]) {
```

Comment:
Can we rename this hook to something more descriptive such as `useScaledWeiAmounts`

Reply:
Let's update it to also receive `toUnit?: EtherUnit`.  If no parameter is passed then it uses `findBestUnit` to get the target unit